causalml/metrics/visualize.py

```diff
@@ -77,11 +77,13 @@ def get_cumlift(
     Returns:
         (pandas.DataFrame): average uplifts of model estimates in cumulative population
     """
-
     assert (
-        (outcome_col in df.columns)
-        and (treatment_col in df.columns)
-        or treatment_effect_col in df.columns
+        (outcome_col in df.columns and df[[outcome_col]].notnull().all().bool())
+        and (treatment_col in df.columns and df[[treatment_col]].notnull().all().bool())
+        or (
+            treatment_effect_col in df.columns
+            and df[[treatment_effect_col]].notnull().all().bool()
```

Comment:
The return type of `all()` is already `bool`. It doesn't need `bool()`.

Reply:
What about adding the `skipna=False` as an input argument so users can explicitly choose to allow NaNs?